src/pdfkit.cs

```diff
@@ -1157,6 +1171,18 @@ interface PdfDocument : NSCopying {
 		[Notification]
 		NSString DidEndPageWriteNotification { get; }
 
+		[iOS (15,0), Mac (12,0), MacCatalyst (15,0)]
+		[Field ("PDFDocumentFoundSelectionKey")]
+		NSString FoundSelectionKey { get; }
+
+		[iOS (15,0), Mac (12,0), MacCatalyst (15,0)]
+		[Field ("PDFDocumentPageIndexKey")]
+		NSString PageIndexKey { get; }
+
+		[iOS (15,0), Mac (12,0), MacCatalyst (15,0)]
+		[Field ("PDFDocumentAccessPermissionsOption")]
+		NSString AccessPermissionsOption { get; }
```

Comment:
that last field should with inside `PdfDocumentWriteOptionKeys`

the _hint_ is that the previous fields (in the headers) are similarly named and (in binding files) already grouped into their own type

Reply:
and also update the `PdfDocumentWriteOptions` string dictionary :)

Reply:
@spouliot I added `PDFDocumentAccessPermissionsOption` to `PdfDocumentWriteOptionKeys` and `PdfDocumentWriteOptions`. I followed along with the pattern of the other elements in there, but not sure why the "+PDFKit" is there or if I needed to add that.

Reply:
Oops forgot to run tests, I'll do that now

Reply:
Okay the tests look good!